### PR TITLE
Actually adds the blood-drunk miner ruins

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: A strange new Resonant Signal has appeared on lavaland.
/:cl:

Three variants on the ruin:
Beast, which is standard.
Guidance, in which the miner heals a tiny amount on attacks and a much larger amount when gibbing a target.
Hunter, in which the miner can dash immediately after attacking.

The ruin itself is just a small cave with an assortment of stone tiles.

### CODE REVIEW

**Comment by Chronitonity** [comment]
pics would still be nice

**Comment by Repukan** [comment]
>Resonant

😢

File: code/datums/ruins/lavaland.dm [hunk 1]
```diff
@@ -123,6 +123,22 @@
 	cost = 0
 	allow_duplicates = FALSE
 
+/datum/map_template/ruin/lavaland/blood_drunk_miner
+	name = "Blood-Drunk Miner"
+	id = "blooddrunk"
+	description = "A strange arrangement of stone tiles and an insane, beastly miner contemplating them."
+	suffix = "lavaland_surface_blooddrunk1.dmm"
+	cost = 0
```
Comment (DoctoreMad) [inline code/datums/ruins/lavaland.dm]: Really?

Comment (ChangelingRain) [inline code/datums/ruins/lavaland.dm]: It'll only spawn once, if at all.

Comment (DoctoreMad) [inline code/datums/ruins/lavaland.dm]: What is the point of saying "if at all" when you're making a PR that will make it always spawn?

Comment (ChangelingRain) [inline code/datums/ruins/lavaland.dm]: Because that's the current condition, smart guy, prs aren't guaranteed.

**Review by Cyberboss — APPROVED** [review]

**Comment by FantasticFwoosh** [comment]
Merged without pictures, what the fuck.

**Comment by Repukan** [comment]
Fwoosh goddamit go look at the original pr

**Comment by FantasticFwoosh** [comment]
Its still courteous since there was no actual pictures of THE RUIN, just the miner on that pr

**Comment by Repukan** [comment]
Who gives a shit about the ruin. If anything your complaint belongs in the original pr, not the fix for it not spawning

**Comment by FantasticFwoosh** [comment]
Dude, i just asked for a picture of the ruin mentioned in this thread that wasn't supplied, stop backpeddling so hard you've entered the nth dimension.

> @ChangelingRain : "The ruin itself is just a small cave with an assortment of stone tiles." 

Remembering necropolis stone tiles are new and aesthetic, and its already been asked to put pictures up.

**Pull request merged**